model/clusters_mgmt/v1/cloud_region_type.model

```diff
@@ -40,4 +40,16 @@ class CloudRegion {
 	// 'true' if the region is supported for Hypershift deployments, 'false' otherwise.
 	SupportsHypershift Boolean
 
+	// Whether the region is an AWS GovCloud region.
+	@json(name = "govcloud")
+	GovCloud Boolean
+
+	// (GCP only) Comma-separated list of KMS location IDs that can be used with this region.
+	// E.g. "global,nam4,us". Order is not guaranteed.
+	KMSLocationID String
```

Comment:
Can the metamodel parse this or should it be `KmsLocationId`?

Reply:
Good question, lemme check... :heavy_check_mark: Yes it parses it fine:

openapi:
```json
          "kms_location_id": {
            "description": "(GCP only) Comma-separated list of KMS location IDs that can be used with this region.\nE.g. \"global,nam4,us\". Order is not guaranteed.",
            "type": "string"
          },
          "kms_location_name": {
            "description": "(GCP only) Comma-separated list of display names corresponding to KMSLocationID.\nE.g. \"Global,nam4 (Iowa, South Carolina, and Oklahoma),US\". Order is not guaranteed but will match KMSLocationID.\nUnfortunately, this API doesn't allow robust splitting - Contact ocm-feedback@redhat.com if you want to rely on this.",
            "type": "string"
          },
```
go:
```go
		stream.WriteObjectField("kms_location_id")
		stream.WriteString(object.kmsLocationID)
             ...
		stream.WriteObjectField("kms_location_name")
		stream.WriteString(object.kmsLocationName)
             ...

// KMSLocationID returns the value of the 'KMS_location_ID' attribute, or
// the zero value of the type if the attribute doesn't have a value.
//
// (GCP only) Comma-separated list of KMS location IDs that can be used with this region.
// E.g. "global,nam4,us". Order is not guaranteed.
func (o *CloudRegion) KMSLocationID() string {
  ...

// GetKMSLocationID returns the value of the 'KMS_location_ID' attribute and
// a flag indicating if the attribute has a value.
//
// (GCP only) Comma-separated list of KMS location IDs that can be used with this region.
// E.g. "global,nam4,us". Order is not guaranteed.
func (o *CloudRegion) GetKMSLocationID() (value string, ok bool) {
  ...
  
// KMSLocationName returns the value of the 'KMS_location_name' attribute, or
// the zero value of the type if the attribute doesn't have a value.
//
// (GCP only) Comma-separated list of display names corresponding to KMSLocationID.
// E.g. "Global,nam4 (Iowa, South Carolina, and Oklahoma),US". Order is not guaranteed but will match KMSLocationID.
// Unfortunately, this API doesn't allow robust splitting - Contact ocm-feedback@redhat.com if you want to rely on this.
func (o *CloudRegion) KMSLocationName() string {
  ...

// GetKMSLocationName returns the value of the 'KMS_location_name' attribute and
// a flag indicating if the attribute has a value.
//
// (GCP only) Comma-separated list of display names corresponding to KMSLocationID.
// E.g. "Global,nam4 (Iowa, South Carolina, and Oklahoma),US". Order is not guaranteed but will match KMSLocationID.
// Unfortunately, this API doesn't allow robust splitting - Contact ocm-feedback@redhat.com if you want to rely on this.
func (o *CloudRegion) GetKMSLocationName() (value string, ok bool) {
  ...

// KMSLocationID sets the value of the 'KMS_location_ID' attribute to the given value.
//
//
func (b *CloudRegionBuilder) KMSLocationID(value string) *CloudRegionBuilder {
  ...

// KMSLocationName sets the value of the 'KMS_location_name' attribute to the given value.
//
//
func (b *CloudRegionBuilder) KMSLocationName(value string) *CloudRegionBuilder {
  ...
```